mantidproject/mantid
# Vesuvio - Additional flag for disabling multiple scattering

Added a flag 'ms_enabled', which can be used in vesuvio user scripts for disabling multiple scattering.

**To test:**
1. Add the line 
```python
flags['ms_enabled'] = False
```
in the example script (supplied below).
2. Open the script in the python editor and execute it.
3. Ensure there is no workspace created containing 'MultipleScattering'.

Example script (**Does not currently work as expected - path to IFP file in this script is specific to machine it was created on**): [user_script_simplified_output_original.zip](https://github.com/mantidproject/mantid/files/1162852/user_script_simplified_output_original.zip)

Fixes #20056. 

**Release Notes** 

Added this feature to the improvements section of the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
